README.md

```diff
@@ -38,7 +38,8 @@ Afterwards you can start the server like this:
 You can also start it via the built-in nREPL CLI:
 
 ``` shell
-$ clj -R:nrepl -m nrepl.cmdline -t fastlane.core/transit+json
+# Change the nREPL and fastlane versions to your preferences
+$ clj -Sdeps '{:deps {nrepl {:mvn/version "0.6.0-SNAPSHOT"} nrepl/fastlane {:mvn/version "0.2.0-SNAPSHOT"}}}' -m nrepl.cmdline --transport fastlane.core/transit+json
```

Comment:
I'd rather use the latest stable nREPL here. 

Perhaps we can also suggest the creation of this `nrepl` profile and also we can mention how to setup fastlane with `lein` and with the new config files.

Reply:
Hunn, good point, gonna check on lein and add these other options to start nREPL

Reply:
Only if you're on the newest lein. I think I added this there.

Reply:
Here it looks correct https://github.com/technomancy/leiningen/blob/master/src/leiningen/repl.clj#L215

Reply:
Yes, I've downloaded `lein`2.8.3, which users `nREPL` 0.5.3.

Is there some `project.clj` with you where it already worked using `lein nREPL` with a local `.nrepl.edn`?

Reply:
I didn't have time for this. I just assumed that this would work, as it seemed fairly simply code-wise. Perhaps I made some mistake, which should be easy to track down if you're inclined to debug the file I linked to.

Reply:
Thanks, gonna try to debug it today o/